tools/metabase-dc.json

```diff
@@ -1,337 +1,340 @@
-"from": "[a-zA-Z0-9]{8}!@#$%^&_[a-zA-Z0-9]{8}",
-"required": true
-},
-{
-"name": "DATABASE_PASSWORD",
-"displayName": "Database Password",
-"description": "A minimum 16 character password that is generated in the target database, and then copied over into this field.",
-"generate": "expression",
-"from": "[a-zA-Z0-9]{16}",
-"required": true
-},
-{
-"name": "EMBEDDING_SECRET_KEY",
-"displayName": "Embedding Secret Key",
-"description": "A key that is used to embed metabase screens from within a website",
-"generate": "expression",
-"from": "[a-zA-Z0-9]{16}",
-"required": true
-}
-]
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "nrpti-metabase"
+  },
+  "objects": [
+    {
+      "kind": "Secret",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-${SUFFIX}-secret",
+        "annotations": {
+          "template.openshift.io/expose-password": "{.data['database-password']}",
+          "as-copy-of": "template.${NAME}-secret"
+        }
+      },
+      "stringData": {
+        "admin-email": "${ADMIN_EMAIL}",
+        "admin-password": "${ADMIN_PASSWORD}",
+        "admin-username": "${NAME}-admin",
+        "database-password": "${DATABASE_PASSWORD}",
+        "embedding-secret-key": "${EMBEDDING_SECRET_KEY}"
+      }
+    },
+    {
+      "kind": "Service",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-${SUFFIX}",
+        "annotations": {
+          "description": "Exposes and load balances the application pods"
+        }
+      },
+      "spec": {
+        "ports": [
+          {
+            "name": "3000-tcp",
+            "protocol": "TCP",
+            "port": 3000,
+            "targetPort": 3000
+          }
+        ],
+        "selector": {
+          "name": "${NAME}-${SUFFIX}"
+        },
+        "type": "ClusterIP",
+        "sessionAffinity": "None"
+      }
+    },
+    {
+      "kind": "Route",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-${SUFFIX}"
+      },
+      "spec": {
+        "host": "${NAME}-${SUFFIX}-${APPLICATION_DOMAIN}",
+        "tls": {
+          "insecureEdgeTerminationPolicy": "Redirect",
+          "termination": "edge"
+        },
+        "path": "/",
+        "to": {
+          "kind": "Service",
+          "name": "${NAME}-${SUFFIX}"
+        },
+        "port": {
+          "targetPort": "3000-tcp"
+        },
+        "wildcardPolicy": "None"
+      }
+    },
+    {
+      "apiVersion": "v1",
+      "kind": "PersistentVolumeClaim",
+      "metadata": {
+        "name": "${NAME}-${SUFFIX}",
+        "annotations": {
+          "volume.beta.kubernetes.io/storage-provisioner": "kubernetes.io/glusterfs"
+        }
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteMany"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "${METABASE_PVC_SIZE}"
+          }
+        }
+      }
+    },
+    {
+      "kind": "ImageStream",
```

Comment:
This `ImageStream` section isn't needed.  All this does is create an empty `ImageStream` object in the prod namespace.

Reply:
Fixed in #af72ccd